application/consumer-kafka-quarkus/src/test/java/webapp/tier/service/KafkaSubscribeServiceTest.java

```diff
@@ -2,22 +2,39 @@
 
 import static org.junit.jupiter.api.Assertions.fail;
 
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.reactive.messaging.Channel;
+import org.eclipse.microprofile.reactive.messaging.Emitter;
 import org.junit.jupiter.api.Test;
 
 import io.quarkus.test.junit.QuarkusTest;
+import io.smallrye.reactive.messaging.annotations.Merge;
 
 @QuarkusTest
 class KafkaSubscribeServiceTest {
 
+	@Inject
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6104857)